.npmignore

```diff
@@ -1,11 +1,22 @@
 node_modules
 bower_components
 data
-examples/_original
-examples/_output
-examples/_diff
-index.html
-script
+docs
+examples
```

Comment:
If we were to import examples to vl-editor, what I did with examples is already correct.